components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/event-simulator/feed_simulator.js

```diff
@@ -2264,17 +2264,35 @@ define(['jquery', 'log', './simulator-rest-client', 'lodash', './open-siddhi-app
     self.removeUnavailableSimulationsFromUi = function (simulations) {
         var activeSimulations = simulations.activeSimulations;
         var simulationName, i;
-        for (i = 0; i < activeSimulations.length; i++) {
-            simulationName = activeSimulations[i].properties.simulationName;
-            if (!(simulationName in self.activeSimulationList)) {
-                self.$eventFeedConfigTabContent.find('div[data-name="' + simulationName + '"]').remove();
+        for (var simulationName in self.activeSimulationList) {
+            if( self.activeSimulationList.hasOwnProperty(simulationName) ) {
+                var foundSimulaion = false;
+                for (i = 0; i < activeSimulations.length; i++) {
+                    if (activeSimulations[i].properties.simulationName == simulationName ) {
+                        foundSimulaion = true;
+                    }
+                }
+                if (foundSimulaion == false) {
```

Comment:
@erangatl , We don't need to check '==' for boolean. simply use below. 

if(! foundSimulaion) {

}

Reply:
Check other places as well.